huaweicloud/resource_huaweicloud_rds_instance_v3.go

```diff
@@ -244,10 +244,18 @@ func resourceRdsInstanceV3() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
 			"created": {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"time_zone": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Default:  "UTC+08:00",
+			},
```

Comment:
suggest to move the` Optional` agrument before `Computed`.
By the way, the `Default` is unnecessary as the API with process it.